pcsx2/gui/ConsoleLogger.h

```diff
@@ -220,3 +223,29 @@ class ConsoleLogFrame : public wxFrame
 
 	void OnLoggingChanged();
 };
+
+template<typename ... Args>
+void OSDlog(ConsoleColors color, bool console, const std::string& format, Args ... args) {
+	if(!GSosdLog) return;
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-security"
```

Comment:
Cheers, you were spot on with the empty arguments thing.  I've incorporated your code into my branch and should have it pushed out soon.